POEApi.Transport/HttpTransport.cs

```diff
@@ -106,6 +126,43 @@ public bool Authenticate(string email, SecureString password, bool useSessionID)
             return true;
         }
 
+        private void CloudFlareSessionIdLogin()
+        {
+            try
+            {
+                using (var clearanceHandler = new ClearanceHandler {MaxRetries = 2})
+                using (var handler = new HttpClientHandler() {CookieContainer = credentialCookies, Proxy = getProxySettings()})
+                {
+                    clearanceHandler.InnerHandler = handler;
+
+                    using (var client = new HttpClient(clearanceHandler))

```

Comment:
This is not the correct way to be using `HttpClient` (creating a new copy for every request; see [here](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/)).  **However,** since by design we're only using the `HttpClient` once per every time we run Procurement, it is okay.  When we switch to only using `HttpClient`, we'll need to be more thorough.

Reply:
I wanted to leave as much alone as possible xD